extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -564,9 +571,7 @@ export class AmpStoryPage extends AMP.BaseElement {
           });
         });
       });
-      this.prefersReducedMotion_()
-        ? this.maybeFinishAnimations_()
-        : this.maybeStartAnimations_();
```

Comment:
This previous worked around likely failed since the browser would reject the jump to `finish` the animations. Instead, we don't start them or apply their styling at all.

Reply:
This is a big problem and completely breaks the Story UX. A lot of Stories have animations like slide/fade in to reveal the text content.
Users with prefers reduced motion wouldn't be able to access the content.

How is the existing code broken? We need to keep the behavior it was trying to have (show animations in their end state).

Reply:
By bypassing the runner we show the content in its end state.
It appears that prefers-reduced-motion was effecting the animation runner and not allowing us to skip to the last frame in this code.

Reply:
See the original disabling of `AnimationManager`, which applies the styles that hide them. By disabling earlier, the content will **always** appear. **The previous workaround caused the problem you describe.**